citeIt/models.py

```diff
@@ -1,5 +1,5 @@
 from django.db import models
-from django.utils.encoding import python_2_unicode_compatible
+from six import python_2_unicode_compatible
```

Comment:
I believe the `python_2_unicode_compatible` was added when we were [supporting both Python 2 and 3](https://github.com/unt-libraries/django-citeit/pull/1). Since Python 2 is long out of support, I think we could remove all references to this.